Controller/ContentTypeController.php

```diff
@@ -130,7 +130,7 @@ public function createContentTypeAction($contentTypeGroupId, $languageCode = nul
             $contentTypeGroup = $this->contentTypeService->loadContentTypeGroup($contentTypeGroupId);
 
             $contentTypeCreateStruct = new ContentTypeCreateStruct([
-                'identifier' => 'new_content_type',
+                'identifier' => '__new__' . md5(microtime(true)),
```

Comment:
given this is UI maybe it would be enough to use `time()` or at least skip md5 so it is a human readable increasing number as opposed to hash.